File: app/src/main/java/com/aar/app/wordsearch/model/GameTheme.java
package com.aar.app.wordsearch.model;

public class GameTheme {

    public static final GameTheme NONE = new GameTheme(-1, "");

    private int mId;
    private String mName;

    public GameTheme(int id, String name) {
        mId = id;
        mName = name;
    }

    public int getId() {
        return mId;
    }

    public void setId(int id) {
        mId = id;
    }

    public String getName() {
        return mName;
    }

    public void setName(String name) {
        mName = name;
    }
}

File: app/src/main/java/com/aar/app/wsp/model/GameTheme.kt
package com.aar.app.wsp.model

import androidx.room.ColumnInfo
import androidx.room.Entity
import androidx.room.PrimaryKey

@Entity(tableName = "game_themes")
class GameTheme @JvmOverloads constructor(
    @ColumnInfo(name = "id")
    @PrimaryKey(autoGenerate = true)
    var id: Int = -1,
    @ColumnInfo(name = "name")
    var name: String = ""
) {
    companion object {
        @JvmField
        val NONE = GameTheme(-1, "")
    }
}